debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -219,12 +220,25 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
                                                       .withInvisibleRecommender()
                                                       .withDescription("The schemas for which events must not be captured");
 
+    public static final Field TIME_PRECISION_MODE = Field.create("time.precision.mode")
+                                                             .withDisplayName("Time Precision")
+                                                             .withEnum(TemporalPrecisionMode.class, TemporalPrecisionMode.ADAPTIVE)
```

Comment:
@jpechane, doesn't this make `adaptive` available to MySQL again and, worse, make it the default for it, too?

Reply:
There is `MySqlConnectorConfig#TIME_PRECISION_MODE` which doesn't contain the value; going forward, we should remove that and use the one from the base, but validate that `adaptive` isn't used (code for that is in place already).

Reply:
@gunnarmorling Fixed, I forgot to override the attribute/method.